src/views/Header.js

```diff
@@ -80,6 +83,24 @@ class Header extends React.Component<void, HeaderProps, void> {
     );
   }
 
+  _tvEventHandler: TVEventHandler;
+
+  componentDidMount(): void {
+    this._tvEventHandler = new TVEventHandler();
+    this._tvEventHandler.enable(this, function(cmp, evt) {
```

Comment:
If I'm understanding this correctly, the first parameter is `this` and second parameter is event listener. Am I right? looks weird